src/t2iapi/combined/combined_requests.proto

```diff
@@ -70,3 +70,14 @@ message SetModeOfOperationAndSetOperatingModeRequest {
                                             // shall be set
     t2iapi.operation.OperatingMode operating_mode = 4; // OperatingMode value to be set for the operation
 }
+
+/*
+Request to set the @ActivationState of the system context with the specified handle to the requested value and to set
+the @ContextAssociation of a context state from the context descriptor with the given handle to the requested value.
```

Comment:
Adjust the wording to match the other context manipulations.

```suggestion
Request to set the @ActivationState of the system context with the specified handle to the requested value and to create or adapt
the @ContextAssociation of a context state from the context descriptor with the given handle to the requested value.
```

Reply:
changed